rules/windows/process_creation/sysmon_apt_sourgrum.yml

```diff
@@ -0,0 +1,43 @@
+title: SOURGUM Actor Behaviours
+id: 7ba08e95-1e0b-40cd-9db5-b980555e42fd
+description: Suspicious behaviours related to an actor tracked by Microsoft as SOURGUM
+author: 
+    - MSTIC
+    - FPT.EagleEye
+status: experimental
+level: high
+references:
+    - sha256=c299063e3eae8ddc15839767e83b9808fd43418dc5a1af7e4f44b97ba53fbd3d
```

Comment:
It is not human readable reference, this IOC is in the microsoft ref, and it is not use in the rule.
I don't think this is helpful in understanding the rule.